__tests__/[id].test.jsx

```diff
@@ -21,7 +25,7 @@ describe('DetailPage', () => {
       <DetailPage fetchedTeamMember={MOCK_TEAM_MEMBER} />,
     );
```

Comment:
Did we want to add a few more assertions here for the other sub-component headers as per stand-up?